mix.exs

```diff
@@ -71,7 +71,7 @@ defmodule Bundlex.Mixfile do
     [
       {:bunch, "~> 1.0"},
       {:qex, "~> 0.5"},
-      {:req, "~> 0.4.0"},
+      {:req, "~> 0.5.0"},
```

Comment:
Since we're only doing a single `Req.get!`, I guess we can relax the requirement to avoid such problems in the future

```suggestion
      {:req, ">= 0.4.0"},
```